### DISCUSSION TITLE
Welcome to yolo Discussions!

### DISCUSSION THREAD

**Discussion by pseudocc** [real-life-copium/yolo#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by main76** [comment]
galaxy-brain!

**Comment by cemrecodes** [comment]
hello

**Reply by pseudocc** [reply]
Hi, If you want to achieve a galaxy-brain badge too. You might need to wait for a few days, I am going to create CI pipelines for it in my spare time, which would take 3-5 days to complete.
